content/en/docs/zero-code/java/agent/extensions.md

```diff
@@ -49,8 +68,1083 @@ extensions:
   existing `Tracer` directly or extend it. As you have your own `Advice`, you
   can control which `Tracer` you use.
 
-## Extension examples
+## Loading an Extension at Runtime
+
+There are two ways to use extensions with the Java agent:
+
+1. **Load at runtime** using the `-Dotel.javaagent.extensions` option (described
```

Comment:
`Load at runtime` is a bit weird, extensions added to agent are also loaded at runtime. Perhaps we shoudl use `As a separate jar file` or something like that.